### DISCUSSION TITLE
Relative paths and directory reads without playlist [Win]

### DISCUSSION THREAD

**Discussion by Lyagva** [savonet/liquidsoap#3095] [User Support]
Win10, Icecast 2.4.4, Liquidsoap 2.1.4

As far as I know, I can just load a folder with audio files like a normal playlist: For example: 
`source = playlist("/music/subfolder/")`
According to this, my music goes in `<current terminal folder>/music/subfolder/`, right?

But it just don't work... I'm getting an error:
    Couldn't read playlist: request resolution failed.
And no other errors like "URI is invalid" or some errors while reading the files

Full path doesn't work either. I use "e:/radio/raw/morning/" and in this state it says URI is invalid. I also tried replacing slashes with backslashes and double backslashes. But none of them are working

Btw, on Ubuntu it works pretty good using "/<path>"

**Comment by toots** [comment]
Hi! Paths starting with `/` are considered absolute paths. For relative paths you should do: `./path/to/file` which would then resolve to `<current directory>/path/to/file`.
